lighthouse-core/scripts/open-devtools.sh

```diff
@@ -19,28 +24,13 @@ if [ -z "${CHROME_PATH:-}" ]; then
   exit 1
 fi
 
-echo "CHROME_PATH: $CHROME_PATH"
-echo "DEVTOOLS_PATH: $DEVTOOLS_PATH"
-
-if [ ! -d "$DEVTOOLS_PATH" ]; then
-  echo "No devtools found at $DEVTOOLS_PATH. Have you run 'yarn test-devtools' yet?"
-  exit 1
+if [ "$DEFAULT_DEVTOOLS_PATH" == "$DEVTOOLS_PATH" ]; then
+  bash "$LH_ROOT/lighthouse-core/test/chromium-web-tests/setup.sh"
```

Comment:
Could this be `source`, then get rid of some of the variables above and make the condition:
`-z "${DEVTOOLS_PATH:-}"`?